examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -29,22 +29,18 @@
 
 import keras_cv
 
-# Temporarily need PyCOCOCallback to verify
-# a 1:1 comparison with the PyMetrics version.
-from keras_cv.callbacks import PyCOCOCallback
-
 low, high = resource.getrlimit(resource.RLIMIT_NOFILE)
 resource.setrlimit(resource.RLIMIT_NOFILE, (high, high))
 
 flags.DEFINE_integer(
     "epochs",
-    35,
+    100,
```

Comment:
I'm shocked going over the same dataset so many times could be productive without an early stopping callback

Reply:
I do have early stopping :)